### PR TITLE
Update CLI docs

### PR DESCRIPTION
**Pull request by hinthornw**

### CODE REVIEW

File: docs/docs/cloud/reference/cli.md [hunk 1]
```diff
@@ -27,101 +42,134 @@ The LangGraph CLI requires a JSON configuration file with the following keys:
...
 **Options**
 
-| Option | Default | Description |
-| ------ | ------- | ----------- |
-| `--platform TEXT` | | Target platform(s) to build the Docker image for. Example: `langgraph build --platform linux/amd64,linux/arm64` |
-| `-t, --tag TEXT` | | **Required**. Tag for the Docker image. Example: `langgraph build -t my-image` |
-| `--pull / --no-pull` | `--pull` | Build with latest remote Docker image. Use `--no-pull` for running the LangGraph Cloud API server with locally built images. |
-| `-c, --config FILE` | `langgraph.json` | Path to configuration file declaring dependencies, graphs and environment variables. |
-| `--help` | | Display command documentation. |
+| Option               | Default          | Description                                                                                                                  |
```
Comment (eyurtsev) [inline docs/docs/cloud/reference/cli.md:93]: thank you

File: docs/docs/cloud/reference/cli.md [hunk 2]
```diff
@@ -27,101 +42,134 @@ The LangGraph CLI requires a JSON configuration file with the following keys:
...
+| `-c, --config FILE`          | `langgraph.json`          | Path to configuration file declaring dependencies, graphs and environment variables.                                  |
+| `-d, --docker-compose FILE`  |                           | Path to docker-compose.yml file with additional services to launch.                                                   |
+| `-p, --port INTEGER`         | `8123`                    | Port to expose. Example: `langgraph test --port 8000`                                                                 |
+| `--pull / --no-pull`         | `pull`                    | Pull latest images. Use --no-pull for running the server with locally-built images. Example: `langgraph up --no-pull` |
+| `--recreate / --no-recreate` | `no-recreate`             | Recreate containers even if their configuration and image haven't changed                                             |
+| `--help`                     |                           | Display command documentation.                                                                                        |
+
+### `dockerfile`
```
Comment (eyurtsev) [inline docs/docs/cloud/reference/cli.md:128]: amazing

File: docs/docs/concepts/langgraph_cli.md [hunk 3]
```diff
@@ -1,3 +1,34 @@
 # LangGraph CLI
 
+The LangGraph CLI is a multi-platform command-line tool for building and running the LangGraph API server locally. It offers an alternative to the [LangGraph Studio desktop app](./langgraph_studio.md) for developing and testing agents. The CLI is supported across all major OS's (Linux, Windows, MacOS).
```
Comment (eyurtsev) [inline docs/docs/concepts/langgraph_cli.md:3]: OS's -> operating systems

File: docs/docs/concepts/langgraph_cli.md [hunk 4]
```diff
@@ -1,3 +1,34 @@
...
+## Installation
+
+The LangGraph CLI can be installed via Homebrew (on macOS) or pip:
+
+=== "Homebrew"
+    ```bash
+    brew install langgraph-cli
+    ```
+
+=== "pip"
```
Comment (eyurtsev) [inline docs/docs/concepts/langgraph_cli.md:14]: is there an npm version?

File: docs/docs/concepts/langgraph_cli.md [hunk 5]
```diff
@@ -1,3 +1,34 @@
...
+    ```bash
+    brew install langgraph-cli
+    ```
+
+=== "pip" 
+    ```bash
+    pip install langgraph-cli
+    ```
+
+## Commands
```
Comment (eyurtsev) [inline docs/docs/concepts/langgraph_cli.md:19]: this is perfect

File: docs/docs/concepts/langgraph_cli.md (hunk 4 above)
Comment (hinthornw) [reply docs/docs/concepts/langgraph_cli.md:14]: no

Comment (hinthornw) [reply docs/docs/concepts/langgraph_cli.md:14]: It works for JS projects already

**Pull request merged**